includes/modules/attributes.php

```diff
@@ -242,7 +242,6 @@
                       break;
                       case '3':
                       $tmp_attributes_image_row++;
-                      //                  if ($tmp_attributes_image_row > PRODUCTS_IMAGES_ATTRIBUTES_PER_ROW) {

```

Comment:
Here, remove the bracket and leave the line. Serves as a reference for an alternate approach.

The others below can be removed.